.github/workflows/ci.yml

```diff
@@ -0,0 +1,19 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup .NET Core SDK
```

Comment:
Since SDK version 5, .NET Core should be just .NET.

Reply:
I fixed this workflow's name.